priv/repo/migrations/20240425131500_add_basic_permissions.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Trento.Repo.Migrations.AddBasicPermissions do
+  use Ecto.Migration
+
+  def up do
+    execute "INSERT INTO abilities(id, name, resource, label, inserted_at, updated_at) VALUES (1, 'all', 'all', 'Permits all operations on all the resources', NOW(), NOW())"
+
+    execute "INSERT INTO abilities(id, name, resource, label, inserted_at, updated_at) VALUES (2, 'all', 'users', 'Permits all operations on user resource', NOW(), NOW())"
+  end
```

Comment:
I wonder what were the considerations to make it necessary to load abilities from the db? For instance, I'm curious why can't there be static abilities, i.e. at the code level? (Maybe I'm thinking too much along a role-based approach, and there is a different/more granular approach that has been agreed here for this.)

Reply:
Because using migrations is really convenient to execute/rollback the actions.
Otherwise, we would need to add seeds/release actions to add them. And the same would happen if you want to remove. You would need to explicitily remove them in the code, which would become error prone

Reply:
I see, thanks for the explanation. My comment was a bit broader, though, on the overall approach of loading/storing abilities in the db in the first place, but I guess maybe that is necessitated due to the access control meta-approach considered (RBAC vs granular ability based, etc.). 
On a somewhat side note: I have been reading a bit into [Cedar](https://github.com/cedar-policy/cedar) recently, and they have some interesting ideas about checking/validating static policies that are decoupled from the application code completely, using some formal methods/tools that give better assurances. It might be interesting to evaluate too, at some point.